src/client.js

```diff
@@ -631,12 +631,10 @@ const fileUploader = (client) => {
                                 };
                                 if (action === "publishIfNeeded") {
                                   try {
-                                    const counter =
-                                      await client.NLWS.xtkCounter.increaseValue(
-                                        { name: 'xtkResource' }
-                                      );
+                                    const UUID = Util.generateUUID();
+
                                     const fileRes = {
-                                      internalName: 'RES' + counter,
+                                      internalName: 'WUI-' + UUID,
```

Comment:
Why change the prefix? WUI hints of a web UI and the SDK is supposend to be generic. Why not keep "RES" or pass the prefix as an option/parameter to the function?

Note that XTK internal names cannot start with a number, so just using a UUID would not always work